src/Tribe/Aggregator/Record/CSV.php

```diff
@@ -92,19 +92,34 @@ public function get_csv_data() {
 
 		$rows    = $importer->do_import_preview();
 
+		/*
+		 * Strip whitespace from the beginning and end of row values
+		 */
+		$formatted_rows = array();
+
+		foreach ( $rows as $row ) {
+			$row              = array_map( 'trim', $row );
+			$formatted_rows[] = $row;
```

Comment:
I think you can take advantage that this `foreach` loop is only a single operation and avoid resigning a value to `row`.

```suggestion
			$formatted_rows[] = array_map( 'trim', $row );
```

Reply:
:+1: Done!